### PR TITLE
Adds tests for header reducer

### PR DESCRIPTION
**Pull request by astr0-4**
Adds prettier-eslint as a dev dependency 
https://www.npmjs.com/package/prettier-eslint

### CODE REVIEW

**Review by jocelynjeffrey — CHANGES_REQUESTED** [review]
hey @astr0-4 - the tests are working but something with the reducer isn't working. The input says [object, object] (screenshot included)

screen shot 2017-07-30 at 8 07 46 pm

**Comment by astr0-4** [comment]
good catch @jocelynjeffrey!!! It should be fixed now

**Review by jprittie — APPROVED** [review]
Just wondering if we should put the test for a component or reducer inside the folder where the component/reducer lives.

File: src/tests/reducers/header.reducer.test.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import {
+  headerReducer,
+  DEFAULT_STATE
+} from "../../redux/reducers/header.reducer"
+import {
+  HEADER_ACTIONS,
```
Comment (jprittie) [inline src/tests/reducers/header.reducer.test.js]: When I run the tests I get 'HEADER_ACTIONS is defined but never used.

**Review by jprittie — COMMENTED** [review]
If I do npm run test a warning comes up for the reducer test.

File: src/tests/reducers/header.reducer.test.js (hunk 1 above)
Comment (jocelynjeffrey) [reply src/tests/reducers/header.reducer.test.js]: @jprittie @astr0-4 I see this warning too - if we're not using HEADER_ACTIONS let's delete it and we're good to go!

**Review by jocelynjeffrey — APPROVED** [review]

**Pull request merged**